.github/workflows/weekly-compress-images.yml

```
@@ -0,0 +1,12 @@
+# Compress images on demand (workflow_dispatch), and at 8pm every Monday (schedule).
+# Open a Pull Request if any images can be compressed.
+name: Weekly Compress Images
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '00 8 * * 1'
+jobs:
+  compress-images:
+    uses: bonitasoft/bonita-documentation-site/.github/workflows/_reusable_nightly_compress_image.yml@master
```

**nitpick:** I have just realized that I didn't do a proper review of https://github.com/bonitasoft/bonita-documentation-site/pull/854
We have inconsistencies in the reusable workflows:
- the workflow should be named "_reusable_compress_images_nightly.yml" for consistency with the one used in PR
- the workflow use the main branch of the `calibreapp/image-actions` action whereas the "PR" workflow use a specific commit

Let's talk in person about this 😸

---

Agree about named and commit for action to use, a PR is open https://github.com/bonitasoft/bonita-documentation-site/pull/859 to adapt it.

---

The PR in the documentation-site repository has been merged
```suggestion
    uses: bonitasoft/bonita-documentation-site/.github/workflows/_reusable_compress_images_nightly.yml@master
```